docs/chapter4-payload-handoff-format.rst

```diff
@@ -29,9 +29,834 @@ architecture, either 32 or 64 bits.
+
+An example showing multiple top-level PCI nodes each with its own bus range:
+
+.. code-block:: dts
+
+   
+   //1st pci-rb in its separate segment of PCIEXBAR 0xC0000000
+   pci-rb0@c0000000 {
+      compatible = "pci";
+      #address-cells = <3>;
+      #size-cells = <2>;
+      bus-range = <0x01 0xdf>;
+      reg = <0x0 0xc0000000 0x1 0x0000000>; // whole ECAM region in this root bridge
+      ranges = <...>;
+      //several root ports
+      //properties
+   };
+
+   // 2nd pci-rb in the shared segment base of PCIEXBAR 0xE0000000
+   pci-rb1@e0000000 {
+      compatible = "pci";
+      #address-cells = <3>;
+      #size-cells = <2>;
+      bus-range = <0x24 0x4b>;
+      reg = <0x0 0xe0000000 0x0 0x8000000>; //whole ECAM region in this root bridge
+
+      //non-reloc/non-prefetch/mmio, child-addr, parent-addr, length
+      ranges = <0x82000000 0x0 0x92000000 0x0 0x92000000 0x0 0x10BC0000
+
+      //non-reloc/non-prefetch/mmio, child-addr, parent-addr, length
+      0x82000000 0x2040 0x00000000 0x2040 0x00000000 0x1 0x40000000
+
+      //non-reloc/32bit/io, child-addr, parent-addr, length
+      0x81000000 0x0 0x4000 0x0 0x4000 0x0 0x2000>;
+
+      //non-reloc/non-prefetch/memory, child-addr, parent-addr, length
+      //indicate rb1 does not support above 4GB DMA
+      dma-ranges = <0x82000000 0x0 0x0 0x0 0x0 0x1 0x0>; // 0 ~ 4GB
+
+      //first root port is B0:D0:F0
+      rootport0@0,0 {
+         #address-cells = <3>;
+         #size-cells = <2>;
+         reg = <...>;
+         /* MMIO, IO resource assigned to this root port that can be consumed by
+         its downstream devices. */
+         ranges; 
+      };
+   };
+
+   /* The 3rd pci-rb in the shared segment base of PCIEXBAR 0xE0000000 (starting
+   from 0xE8000000) */
+   pci-rb2@e8000000 {
+      compatible = "pci";
+      #address-cells = <3>;
+      #size-cells = <2>;
+      bus-range = <0x81 0xc8>;
+      reg = <0x0 0xe8000000 0x0 0x8000000>; //whole ECAM region in this root bridge
+      //MMIO, IO resource owned by this root bridge
+      ranges = <...>;
+
+      //non-reloc/non-prefetch/memory, child-addr, parent-addr, length
+      //indicate rb2 does not support above 4GB DMA
+      dma-ranges = <0x82000000 0x0 0x0 0x0 0x0 0x1 0x0>; // 0 ~ 4GB
+
+      //first root port is B128: D0: F0
+      rootport0@0,0 {
+         reg = <...>;
+         /* MMIO, IO resource assigned to this root port that can be consumed by
+         its downstream devices. */
+         ranges;
+      };
+   };
+
+
+4.2.6 Node: /isa (for IO Bus only) (O)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are different kinds of buses each using their own addressing scheme. Among
+them are usually memory addresses, legacy I/O Ports and PCIe. In summary
+one needs to surround the node using legacy I/O ports with a “isa” bus node.
```

Comment:
Could we support the newer usage which are the replacements of isa?
The example FDT node is as below:
    pci-rb {
        espi@1F,0 {
            compatible = "isa";
            serial@3F8 {
                compatible = "isa";
                reg-io-width = <0x00000001>;
                reg = <0x00000001 0x000003f8 0x00000008>;
                current-speed = <0x0001c200>;
            };
        };
    };



spec for lpc and espi:
lpc:
https://www.intel.com/content/dam/www/program/design/us/en/documents/low-pin-count-interface-specification.pdf

espi:
https://www.intel.com/content/dam/support/us/en/documents/software/chipset-software/327432-004_espi_base_specification_rev1.0_cb.pdf

Reply:
Basing on offline discussion, we prefer to have spec only mentions isa node, so even HW side the serial port device is under lpc or espi bus, the DT node will be still isa.

We may re-visit this area later if any data shows lpc node or espi node is necessary.